src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToControlGeneratedAssemblyInfo.cs

```diff
@@ -758,5 +759,58 @@ public void It_does_not_write_to_undefined_assembly_metadata_attribute(string ta
                 AssemblyInfo.Get(assemblyPath).ContainsKey("AssemblyMetadataAttribute").Should().Be(false);
             }
         }
+
+        [Theory]
+        [InlineData("netcoreapp3.1", ".NET Core 3.1")]
+        [InlineData("netcoreapp2.1", ".NET Core 2.1")]
+        [InlineData("netstandard2.1", ".NET Standard 2.1")]
+        [InlineData("net5.0", ".NET 5.0")]
+        public void CheckTargetFrameworkDisplayName(string targetFrameworkVersion, string expectedVersion)
```

Comment:
should `expectedVersion` be `expectedFrameworkDisplayName` based on the attribute?